app/users/user.service.ts

```diff
@@ -89,10 +95,10 @@ export class UserService {
    * @param role If specified, will only return users of that role.
    * @param query An additional string to search for in the result set. If specified, only matches will be returned.
    */
-  public queryAll(page?: number, pageSize = 10, role?: string, query?: string): Promise<{items: User[], metadata: any}> {
+  public queryAll(page?: number, pageSize = 10, roles?: string, query?: string): Promise<{ items: User[], total: any }> {
```

Comment:
Doesn't the type of of the `roles` parameter need to be `string[]` instead of just `string`?

Reply:
already initialized it as a string array so no need to mention string[]..i guess...